ui/src/app/pages/workspace/workspace-about.tsx

```diff
@@ -240,7 +240,7 @@ export const WorkspaceAbout = fp.flow(withUserProfile(), withUrlParams(), withCd
           <div style={styles.infoBox} data-test-id='cdrVersion'>
             <div style={styles.infoBoxHeader}>Dataset</div>
             <div style={{fontSize: '0.5rem'}}>
-              {workspace ? getCdrVersion(workspace, cdrVersionListResponse).name : 'Loading...'}
+              {(workspace && cdrVersionListResponse) ? getCdrVersion(workspace, cdrVersionListResponse).name : 'Loading...'}
```

Comment:
Hm.. this shouldn't be possible. `withCdrVersions()` doesn't render the wrapped component until cdrVersionListResponse is available. I would expect that to work the same for functional components as well, but perhaps I'm missing something.

Reply:
I did not actually observe the problem here.  I can remove this change if you're confident.